[2024-12-22T16:16:10.951+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-22T16:16:11.060+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_top_best_sellers_books.create_books_table manual__2024-12-22T14:55:10.356955+00:00 [queued]>
[2024-12-22T16:16:11.114+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_top_best_sellers_books.create_books_table manual__2024-12-22T14:55:10.356955+00:00 [queued]>
[2024-12-22T16:16:11.117+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 4
[2024-12-22T16:16:11.182+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_books_table> on 2024-12-22 14:55:10.356955+00:00
[2024-12-22T16:16:11.213+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1054) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-22T16:16:11.226+0000] {standard_task_runner.py:72} INFO - Started process 1056 to run task
[2024-12-22T16:16:11.224+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_top_best_sellers_books', 'create_books_table', 'manual__2024-12-22T14:55:10.356955+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpezwc57jp']
[2024-12-22T16:16:11.246+0000] {standard_task_runner.py:105} INFO - Job 44: Subtask create_books_table
[2024-12-22T16:16:12.232+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_top_best_sellers_books.create_books_table manual__2024-12-22T14:55:10.356955+00:00 [running]> on host c50092bbd7fa
[2024-12-22T16:16:13.441+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_top_best_sellers_books' AIRFLOW_CTX_TASK_ID='create_books_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-22T14:55:10.356955+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-22T14:55:10.356955+00:00'
[2024-12-22T16:16:13.452+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-22T16:16:13.561+0000] {sql.py:278} INFO - Executing: CREATE TABLE IF NOT EXISTS books (
    book_id SERIAL PRIMARY KEY,
    author_id BIGINT REFERENCES author_dim(author_id),
    date_id BIGINT REFERENCES date_dim(date_id),
    price NUMERIC,
    rating VARCHAR,
    title VARCHAR,
    image VARCHAR
);
[2024-12-22T16:16:13.607+0000] {base.py:84} INFO - Retrieving connection 'books_connection'
[2024-12-22T16:16:13.664+0000] {base.py:84} INFO - Retrieving connection 'books_connection'
[2024-12-22T16:16:13.692+0000] {sql.py:553} INFO - Running statement: CREATE TABLE IF NOT EXISTS books (
    book_id SERIAL PRIMARY KEY,
    author_id BIGINT REFERENCES author_dim(author_id),
    date_id BIGINT REFERENCES date_dim(date_id),
    price NUMERIC,
    rating VARCHAR,
    title VARCHAR,
    image VARCHAR
);, parameters: None
[2024-12-22T16:16:13.781+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-22T16:16:13.786+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_top_best_sellers_books, task_id=create_books_table, run_id=manual__2024-12-22T14:55:10.356955+00:00, execution_date=20241222T145510, start_date=20241222T161611, end_date=20241222T161613
[2024-12-22T16:16:13.882+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-22T16:16:15.024+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-22T16:16:15.036+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
